libs/coin-evm/src/logic.ts

```diff
@@ -199,7 +200,9 @@ export const getSyncHash = (
     .map(token => token.id + token.contractAddress + token.name + token.ticker)
     .join("");
   const isNftSupported = isNFTActive(currency);
-  const { node = {}, explorer = {} } = currency.ethereumLikeInfo || {};
+
+  const config = getCoinConfig(currency).info;
+  const { node = {}, explorer = {} } = config;
```

Comment:
Do you think we still need this default value? Otherwise we can clean it at the same time.

Reply:
Yes the default values are still needed because node and explorer are not in the config for every evm currency. However we use the objects line 219 to create the hash